std_msgs/Header header
float64 robot_id
geometry_msgs/Pose target_pose